src/diffusers/pipelines/cogview3/pipeline_cogview3plus.py

```diff
@@ -333,7 +333,7 @@ def prepare_extra_step_kwargs(self, generator, eta):
             extra_step_kwargs["generator"] = generator
         return extra_step_kwargs
 
-    # Copied from diffusers.pipelines.latte.pipeline_latte.LattePipeline.check_inputs
+    # Copied from diffusers.pipelines.deprecated.latte.pipeline_latte.LattePipeline.check_inputs
```

Comment:
Not sure how much usage does this pipeline have but maybe we could move this one to deprecated too in view of CogView4?

Reply:
wdyt @a-r-r-o-w?